tests/test_time_series_graph.py

```diff
@@ -693,6 +693,9 @@ def test_get_minimal_graph(self):
         self.assertTrue(min_tsgraph.edge_exists('b lag(n=1)', 'c'))
         self.assertEqual(len(min_tsgraph.nodes), 2)
 
+        # test it does not fail with a very big graph due to recursion
+        self.tsdag_1.extend_graph(forward_steps=100, backward_steps=100).get_minimal_graph()
```

Comment:
is this expensive?

Reply:
I updated it to 200 and to make sure I tasted with the main branch and it would fail :)

Reply:
Fantastic, thanks!